### ISSUE TITLE
Incompatibility with Arclight

### ISSUE THREAD

**Issue by ahdg6** [RedstoneDubstep/Vanishmod#7]
Create:https://github.com/Creators-of-Create/Create

Crash repo:https://paste.ubuntu.com/p/FSkkpjYhb4/

**Comment by RedstoneDubstep** [comment]
Hi, thanks for your report! During my testing on a Forge server, I found out that Create is perfectly compatible with the Vanishmod, however (if I read the crash report correctly) you seem to have issues while running an Arclight server, and the Vanishmod is, in fact, incompatible with such a server. I have no idea why Create is mentioned in the failure message, from what I've seen it's unrelated to the Vanishmod not starting on Arclight.
The issue here is that Arclight redirects (i.e. modifies) a method via Mixin to correctly display join/leave messages, but that same method needs to be redirected by the Vanishmod to not show join/leave messages for vanished players. Mixin doesn't like it if one method is modified twice, and thus crashes. I don't think I can evade that crash because that method redirect from the Vanishmod is mandatory, so unfortunately, the Vanishmod is incompatible with Arclight.
But Arclight is a Bukkit-Forge-Hybrid-Server (if I understood it correctly), and you're surely gonna have no problems finding a Bukkit plugin that achieves the same effect as this Forge mod.

**Comment by ahdg6** [comment]
Thank you for your clarification.

> But Arclight is a Bukkit-Forge-Hybrid-Server (if I understood it correctly), and you're surely gonna have no problems finding a Bukkit plugin that achieves the same effect as this Forge mod.

Most Bukkit plugins do Not support mods, and thus they can Not blocked sound of mods' chest.
That's why I indeed need this mod.

Is there any chances to avoid crashing?

**Comment by RedstoneDubstep** [comment]
I already tried different approaches to evade the crash, for example increasing the priority of my mixins, but that didn't work. I also tried if it's possible to not modify the method redirected by Arclight while still keeping the feature of suppressing join/leave messages, though I couldn't find a way how. And I don't want to dismiss the message suppressing feature entirely, as I consider it a key part of this mod.
Regarding sounds blocking: This mod only suppresses sounds that are directly related to the player, and thus only a minority of sounds actually get suppressed (for example, the sound of a normal chest opening would not get suppressed, even if a vanished player did so). So I suppose that most Vanish plugins out there are better than this mod, if only because they don't crash.
